src/test/php/lang/ast/unittest/parse/BlocksTest.class.php

```diff
@@ -5,7 +5,7 @@ class BlocksTest extends ParseTest {
 
   /** @return void */
   public function setUp() {
-    $this->block= [['(' => [['block' => 'block'], []]]];
+    $this->block= [['(' => [['(name)' => 'block'], []]]];
```

Comment:
Changing the symbols requires changes to the test suite like this one here. These changes do not induce any functional change!